pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${bdg-formats.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.bdgenomics.qc-metrics</groupId>
+      <artifactId>qc-metrics-core</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
I did run into some issues yesterday where command-line maven & IntelliJ got out of sync. IntelliJ seems to have figured things out since yesterday, though there's nothing specific I can point to that I did which fixed it:
![screen shot 2015-07-01 at 2 59 30 pm](https://cloud.githubusercontent.com/assets/98301/8462419/cf64b9dc-2001-11e5-9184-0c4d95b5f320.png)